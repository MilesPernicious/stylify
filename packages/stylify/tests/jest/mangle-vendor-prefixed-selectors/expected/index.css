._b99lu7{
	-webkit-box-orient: vertical
}
._s93ul{
	-webkit-line-clamp: 3
}
._r8rd4{
	-apple-pay-button-type: plain
}
._hob8{
	-apple-trailing-word: auto
}
._2cdb{
	-moz-user-modify: read-only
}
._80pl5n{
	-moz-user-select: auto
}
