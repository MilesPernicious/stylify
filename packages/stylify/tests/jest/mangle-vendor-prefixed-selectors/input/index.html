<div class="-webkit-box-orient:vertical -webkit-line-clamp:3"></div>
<div n:class="'-moz-user-modify:read-only', '-moz-user-select:auto'"></div>
<div v-bind:class="{ '-apple-pay-button-type:plain': true, '-apple-trailing-word:auto'}"></div>
<div class="{if true}-webkit-box-orient:vertical{else}-webkit-line-clamp:3{/if}"></div>
