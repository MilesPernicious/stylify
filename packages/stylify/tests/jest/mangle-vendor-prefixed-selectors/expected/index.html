<div class="_b99lu7 _s93ul"></div>
<div n:class="'_2cdb', '_80pl5n'"></div>
<div v-bind:class="{ '_r8rd4': true, '_hob8'}"></div>
<div class="{if true}_b99lu7{else}_s93ul{/if}"></div>
